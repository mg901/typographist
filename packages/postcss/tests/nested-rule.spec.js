const run = require('./run');

test('transform nested rule', async () => {
  const source = `
        .test,
        .some-test {
          font-size: 1rem;
          &__wrap,
          &__inner {
            font-size: 2rem;
            &:hover {
              background-color: pink;
            }
          }
        }`;

  expect(await run(source)).toMatchSnapshot();
});
