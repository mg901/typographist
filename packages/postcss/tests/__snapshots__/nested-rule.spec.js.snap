// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transform nested rule 1`] = `
.test,
.some-test {
  font-size: 1rem;
}
.test__wrap,
.test__inner,
.some-test__wrap,
.some-test__inner {
  font-size: 2rem;
}
.test__wrap:hover,
.test__inner:hover,
.some-test__wrap:hover,
.some-test__inner:hover {
  background-color: pink;
}
`;
